lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -112,9 +109,6 @@ def settings_to_import_pod_targets
               build_settings['HEADER_SEARCH_PATHS'] = '$(inherited) ' + XCConfigHelper.quote(library_header_search_paths)
               build_settings['OTHER_CFLAGS'] += ' ' + XCConfigHelper.quote(library_header_search_paths, '-isystem')
             end
-            if pod_targets.any? { |t| t.should_build? && t.scoped? }
-              build_settings['FRAMEWORK_SEARCH_PATHS'] = '$PODS_FRAMEWORK_BUILD_PATH'
```

Comment:
Removing this here would likely break search path inheritance.
@segiddins: Can you confirm that?